finetune/config.py

```diff
@@ -322,6 +322,17 @@ def get_default_config():
         bert_use_pooler=True,
         bert_use_type_embed=True,
         #
+        # LongDocBERT variants only
+        max_num_chunks=512,     # Must be set to an integer (not None)
+        max_chunk_length=None,  # Will be set dynamically according to max_length and max_num_chunks
+        chunk_pool_fn="mean",   # aggregation function across chunks in LongDocBERT featurizer
+        batch_size_scaler=2,    # Scaling factor for batch size when passing chunks to BERT featurizer
```

Comment:
Personally I think you can just keep this as batch_size and max_chunk_length.  If you're manually configuring both batch_size and batch_size_scaler, you might as well just set batch_size.

Reply:
That's a good point if there are no other operations where a large batch_size is problematic. @benleetownsend I think the "scaling" of the batch_size to parallelize multiple chunks through the `map_fn` simultaneously was an idea you had, do you have any thoughts?

Using just the batch_size would simplify the logic some.

Reply:
OK after looking at this a second time I understand the motivation better.  The batch size appropriate for the aggregation_op is not the same as the batch size appropriate for the embedding op.  

An alternate suggestion would be:
- Have batch_size mean the batch_size of the aggregation op.
- Have embedding_batch_size mean the batch size of the embedding op.

This feels simpler to me than expressing embedding_batch_size in terms of a fraction of the aggregation_op batch size?

Reply:
Yea that would work as well. The only requirement would be that the `embedding_batch_size` needs to be divisible by `batch_size`. Because you reshape from `[batch_size, num_chunks, sequence_length]` to [embedding_batch_size, num_chunks * batch_size / embedding_batch_size, sequence_length]`.

I guess technically the requirement then becomes that `num_chunks * batch_size` must be divisible by `embedding_batch_size`, but that's getting more complicated

Reply:
I think the code is slightly simpler if the featurizer batch size is a multiple of the overall batch size. But other than this I'm not tracking anything.